src/progress_bar_exercise/ProgressBar/progress-bar.component.js

```diff
@@ -0,0 +1,41 @@
+import { useCss } from "kremling";
+import React, { useEffect, useState } from "react";
+
+export default function ProgressBar({ loading }) {
+  const [seconds, setSeconds] = useState(0)
+  const [opacity, setOpacity] = useState(1)
+
+  useEffect(() => {
+    if (seconds && !loading) { // loading was toggled to false and we have a count going
+      setSeconds(16)
+      const timer = setTimeout(() => setOpacity(0), 3000)
+      return () => clearTimeout(timer)
```

Comment:
In this first condition, bumping seconds to 16 will result in a 100% width (line 25). Additionally, we will trigger opacity to hide the progress bar after 3 seconds